Userland/Utilities/icc.cpp

```diff
@@ -125,7 +126,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
         tag_data_to_first_signature.set(tag_data, tag_signature);
 
-        if (tag_data->type() == Gfx::ICC::CurveTagData::Type) {
+        if (tag_data->type() == Gfx::ICC::CicpTagData::Type) {
+            auto& cicp = static_cast<Gfx::ICC::CicpTagData&>(*tag_data);
+            outln("    color primaries: {} - {}", cicp.color_primaries(),
+                Video::color_primaries_to_string((Video::ColorPrimaries)cicp.color_primaries()));
```

Comment:
This feels like something that should live in LibGfx instead if ICC profiles use it as well, and then LibVideo would use that (for later).

Reply:
https://github.com/SerenityOS/serenity/pull/17369#issuecomment-1422640125 – agreed!

Reply:
I concur!